### PR TITLE
docs: refactor access flow page

### PR DESCRIPTION
**Pull request by cristianmtr**
# refactor the "access flow" page

To follow the structure in https://github.com/jina-ai/internal-tasks/issues/401

### CODE REVIEW

**Review by JohannesMessner — CHANGES_REQUESTED** [review]
In the websocket section there is a slightly passive aggressive (imho) bullet point about not using gRPC if the user is lazy. Maybe instead of this we can have a very short section about gRPC (on the same level with http, graphql, websocket) that says that in principle you could use it from any langauge by compiling jina.proto, but that we only actively support our python client for this or something like that?

File: docs/fundamentals/flow/access-flow-api.md [hunk 1]
```diff
@@ -1,134 +1,45 @@
...
 ```{admonition} Available Protocols
 :class: caution
-Jina Flows can use one of three protocols: gRPC, HTTP, or Websocket. Only Flows that use HTTP can be accessed via the
-methods described below.
+Jina Flows can use one of {ref}`three protocols <flow-protocol>`: gRPC, HTTP, or Websocket. 
+Only Flows that use HTTP can be accessed via the methods described below.
 ```
 
-Outside of using the Jina Client, various forms of sending HTTP requests are the most common way of interacting with a Flow.
+Apart from using the {ref}`Jina Client <client>`, the most common way of interacting way with your deployed Flow is via HTTP.
```
Comment (samsja) [inline docs/fundamentals/flow/access-flow-api.md:23]:
```suggestion
Apart from using the {ref}`Jina Client <client>`, the most common way of interacting with your deployed Flow is via HTTP.
```

**Review by samsja — COMMENTED** [review]
the graphql endpoint is wrongly formatted in that part of the docs. You could maybe fix it with that PR
Screenshot from 2022-04-04 08-55-22

File: docs/fundamentals/flow/access-flow-api.md [hunk 2]
```diff
@@ -178,8 +89,139 @@ curl --request POST 'http://localhost:12345/post' --header 'Content-Type: applic
...
+
+`````
+
+### Arguments
+
+Your HTTP request can include the following parameters:
+
+| Name             | Type         | Description                                                                            | Example                                           |
+| ---------------- | ------------ | -------------------------------------------------------------------------------------- | ------------------------------------------------- |
+| `execEndpoint`   | **required** | Executor {ref}`endpoint <executor-requests>` to target                                 | `"execEndpoint": "/index"`                        |
```
[the remark this answers is not in the document]
Comment (cristianmtr) [reply docs/fundamentals/flow/access-flow-api.md]: Which link? I think this was an older version

File: docs/fundamentals/flow/access-flow-api.md [hunk 3]
```diff
@@ -1,134 +1,45 @@
...
-```{code-block} python
-from docarray import Document, DocumentArray
-from jina import Client
-
-# Flow exposed on host HOST and port PORT
-c = Client(host=HOST, port=PORT)
-response_docs = c.post(on='/search', inputs=DocumentArray())
-```
-````
+This page is about accessing the Flow with external clients. You can also use the Jina Client, provided by us in the `jina` package.
```
[the remark this answers is not in the document]
Comment (cristianmtr) [reply docs/fundamentals/flow/access-flow-api.md]: added

File: docs/fundamentals/flow/access-flow-api.md (hunk 1 above)
Comment (cristianmtr) [reply docs/fundamentals/flow/access-flow-api.md]: done

**Comment by cristianmtr** [comment]
> the graphql endpoint is wrongly formatted in that part of the docs. You could maybe fix it with that PR Screenshot from 2022-04-04 08-55-22

Which part?

**Comment by samsja** [comment]
> > the graphql endpoint is wrongly formatted in that part of the docs. You could maybe fix it with that PR Screenshot from 2022-04-04 08-55-22
> 
> Which part?

the gateway which is in purple is wrongly formatted ( the purple come from my mouse when I did the screenshot)

**Comment by cristianmtr** [comment]
> > > the graphql endpoint is wrongly formatted in that part of the docs. You could maybe fix it with that PR Screenshot from 2022-04-04 08-55-22
> > 
> > 
> > Which part?
> 
> the gateway which is in purple is wrongly formatted ( the purple come from my mouse when I did the screenshot)

So you wanted it to be between backticks, right? Like so 

image

**Review by samsja — APPROVED** [review]

**Review by JohannesMessner — APPROVED** [review]

**Pull request merged**